stdlib/experimental/promql/promql_quantile_neg_test.flux

```diff
@@ -0,0 +1,44 @@
+package promql_test
```

Comment:
promql tests can go with the other tests in the `stdlib/testing/testdata` folder.

Reply:
True. I'll update that